link_analysis/api_module.py

```diff
@@ -349,8 +351,33 @@ def start_process_with(
                                         visualizerParameters[2])
 # end of start_process_with---------------------------------------------
 
+
+def get_all_links_from_all_headers(
+        sendRequestToUpdatingHeadersInBaseFromSite=False,
+        whichSupertypeUpdateFromSite=False):
+
+    print("Started to getting Headers from web_crawler.")
+    start_time = time.time()
+    jsonHeaders = wc_interface.get_all_headers(
+            sendRequestToUpdatingHeadersInBaseFromSite,
+            whichSupertypeUpdateFromSite)
+    print("Finished getting Headers from web_crawler. "
+          f"It spent {time.time()-start_time} seconds.")
+
+    if not jsonHeaders:
```

Comment:
Это проверка на None? Тогда давай лучше  по pep8
```suggestion
    if jsonHeaders is not None:
```

Reply:
Я сам про это говорил, сам забыл. Но в данном месте if jsonHeaders is None:

Reply:
Поправил в коммите.